package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "prop-types": "^15.7.2",
```

Comment:
![image](https://user-images.githubusercontent.com/31897434/76875582-086a5180-6871-11ea-8da1-6e2bf375cce0.png)
You missed this 👆 
Kindly add it and re-submit for review.

Reply:
All necessary changes made. My partner and I are working together, this was just a very short milestone.